docs/articles/remoting/transports.md

```diff
@@ -13,6 +13,12 @@ In this section we'll expand a bit more on what transports are and how Akka.Remo
 ## What are Transports?
 Transports in Akka.Remote are abstractions on top of actual network transports, such as TCP and UDP sockets, and in truth transports have pretty simple requirements.
 
+> [!WARNING]
+> Due to a bug in the remoting serializer, you __can not__ mix v1.4.20 - v1.4.25 systems with __older__
```

Comment:
Again, this oversells the issue - it makes it sound like the entire remoting system is torched, not one particular serializer.